libraries/cmake/source/libpcap/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2014-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed in accordance with the terms specified in
+# the LICENSE file found in the root directory of this source tree.
+
+function(libPcapMain)
+  execute_process(COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/src/configure")
```

Comment:
The configure script should be launched manually on the reference distribution (we are using Ubuntu 16.04), and the output should be replicated in pure CMake

Reply:
@alessandrogario, sorry if I could not understand this. Are you asking me to run the configure and make command manually and generate the cpp files that are needed, and checkin the generated files to some directory and include that generated CPP file in CMake?